### DISCUSSION TITLE
too many emails from github

### DISCUSSION THREAD

**Discussion by Harbingerhq** [community/community#143902] [Actions] [answered]
hi I just signed up recently my email inbox is full of emails from github 
no matter how much I unsubscribe its coming forever how do I stop this I tried Emails Unsubscribe from all topics.   but no luck ![Screenshot_2024-11-08-00-30-00-220_ch.protonmail.android.jpg](https://github.com/user-attachments/assets/5f329890-0867-4204-b017-517ed92952bd)

**Comment by dalvarez2596** [comment, marked as answer]
Hi @Harbingerhq,
I think you could go to your notifications setup :

1. Click on the notifications button at the top right corner: 
<img width="1119" alt="Screenshot 2024-11-07 at 4 09 42 PM" src="https://github.com/user-attachments/assets/515ed689-bf35-4bdb-ab04-a2e8323dd2d9">

2. Go to manage my notifications :

<img width="273" alt="Screenshot 2024-11-07 at 4 10 29 PM" src="https://github.com/user-attachments/assets/96a14a02-6b57-42e7-a986-54cb1cf71b9c">

3. Once there you can select when and how receive the notifications.

For further reference and more about notifications refer to the docs :
[Configuring notifications](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications)

**Reply by Harbingerhq** [reply]
thanks i just wasn't familiar with site the options was  on bottom (I'm on phone)

**Reply by dalvarez2596** [reply]
Sure, if you had any other issues just let me know :) , yeah on the phone looks different hope you find the option.
